src/components/App.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from "react";
+import {
+  BrowserRouter as Router,
+  Route,
+  Routes,
+  Link,
+  useParams,
+} from "react-router-dom";
+import logo from "../logo.png";
+import "./App.css";
+import axios from "axios";
+import CommentForm from "./CommentForm";
+import SightingForm from "./SightingForm";
+
+const App = () => {
+  const [sightings, setSightings] = useState([]);
+
+  const displaySightings = async () => {
+    try {
+      const res = await axios.get("http://localhost:3000/sightings/");
+      console.log(res.data);
+      setSightings(res.data);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    displaySightings();
+  }, []);
+
+  return (
+    <Router>
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <Routes>
+            <Route
+              path="/"
+              element={
+                <div>
+                  <SightingForm
+                    displaySightings={displaySightings}
+                  ></SightingForm>
+                  {sightings.map((sighting) => (
+                    <Link key={sighting.id} to={`/sightings/${sighting.id}`}>
+                      {sighting.id}: {sighting.createdAt}
+                      <br></br>
+                    </Link>
+                  ))}
+                </div>
+              }
+            />
+            {sightings.map((sighting) => (
+              <Route
+                key={sighting.id}
+                path={`/sightings/:sightingId`}
+                element={<Sighting />}
+              />
+            ))}
+          </Routes>
+        </header>
+      </div>
+    </Router>
+  );
+};
+
+const Sighting = () => {
+  const { sightingId } = useParams(); // Fix: Change 'sightingIndex' to 'sightingId'
```

Comment:
comments like such will never be done usually. Here you did it, but didn't remove the comment either. So either don't use such comments, or at least remove if done